docs/web/examples/fortune.rpy

```diff
@@ -38,12 +39,13 @@ def render_GET(self, request):
         Use L{utils.getProcessOutput}, which spawns a process and returns a
         Deferred which fires with its output.
         """
-        request.write("<pre>\n")
+        request.write(b"<pre>\n")
         deferred = utils.getProcessOutput(self.fortune)
-        deferred.addCallback(lambda s:
-                             (request.write(s+"\n"), request.finish()))
-        deferred.addErrback(lambda s:
-                     (request.write(str(s)), request.finish()))
+        deferred.addCallback(lambda s: (request.write(s + b"\n"), request.finish()))
```

Comment:
Man, this is a gross hack. Not that you should fix it now, I just couldn't let it pass.